src/levels.rs

```diff
@@ -259,7 +259,15 @@ pub union Av1Block_intra_inter {
     pub c2rust_unnamed_0: Av1Block_inter,
 }
 
-#[derive(Copy, Clone)]
+impl Default for Av1Block_intra_inter {
```

Comment:
Can we derive it in all cases instead of any manual impls?

Reply:
We can't derive `Default` for unions as far as I can tell, so at least here we need a smol manual impl.

Reply:
Ahh, that's right.  How do we know which variant should be initialized for `Default` then?  How does `c2rust` know?